locales/en/translations.json

```diff
@@ -1,4 +1,18 @@
 {
     "startTitle" : "This is Limited Partnerships",
-    "errorStartAddressMissing" : "Please enter an address"
+    "errorStartAddressMissing" : "Please enter an address",
+
+    "pageNotFound_Title" : "Page not found",
+    "pageNotFound_Text1" : "If you typed the web address, check it is spelled correctly.",
```

Comment:
the key must be explicit to ensure that no errors can be made
we can't use numbers because it will quickly become complicated when one is moved or deleted

"pageNotFound": {
     "spelling": "If you typed the web address, check it is spelled correctly.",
     ...
}